data/json/items/tool/cooking.json

```diff
@@ -125,15 +125,38 @@
     "price_postapoc": 2000,
     "to_hit": -3,
     "bashing": 3,
-    "material": [ "wood", "plastic" ],
+    "material": [ "plastic" ],
     "symbol": ";",
     "color": "light_gray",
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 150 } } ],
-    "initial_charges": 150,
-    "max_charges": 150,
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 75 } } ],
+    "initial_charges": 75,
+    "max_charges": 75,
```

Comment:
#50035 modifies the crafting recipe to require 150 charcoal instead.

Reply:
I've already modified the cooking item Item to have 75 charges.

At least I think I did...